CONTRIBUTING.md

```diff
@@ -201,10 +178,12 @@ make build
 or
 
 ```bash
-npm run make:build
+npm run make:deploy
```

Comment:
@lucferbux , shouldn't it be `npm run make:build` ?

Reply:
@pnaik1 Yes that's right, thanks a lot!